src/modules/create_activity/app/create_activity_usecase.py

```diff
@@ -49,6 +50,20 @@ def __call__(self, code: str, title: str, description: str, activity_type: ACTIV
         if len(responsible_professors) != len(responsible_professors_user_id):
             raise NoItemsFound("responsible_professors")
 
+        if (delivery_model == DELIVERY_MODEL.ONLINE or delivery_model == DELIVERY_MODEL.HYBRID) and link is None:
```

Comment:
faz esses dois ifs que vc levantou NoItemsFound no controller e levanta MISSINGPARAMETERS

Reply:
e trata eles no controller tipo se a mensagem for "place" vc coloca "Local da atividade ausente"